app/src/main/res/xml/preferences_server.xml

```diff
@@ -8,7 +8,7 @@
         app:isPreferenceVisible="false"/>
     <Preference
         android:key="activate_server_hint"
-        android:summary="@string/server_activate_swipe_hint"
+        android:summary="@string/server_activate_gesture_hint"
```

Comment:
It's going to break all the existing translations for this key no?

Reply:
Yes, that is intentional to avoid other languages still having the string with three finger swipe (because of outdated translations), which may be incorrect.